test/js_test.go

```diff
@@ -8709,6 +8717,9 @@ func TestJetStreamDirectGetMsg(t *testing.T) {
 }
 
 func TestJetStreamConsumerReplicasOption(t *testing.T) {
+	s := RunBasicJetStreamServer()
+	defer shutdownJSServerAndRemoveStorage(t, s)
```

Comment:
need an extra JS server created here? I think it would not be part of the cluster?

Reply:
I missed that when cleaning up after experiments...